extractor.rb

```diff
@@ -0,0 +1,6 @@
+class Extractor
+  def method_that_does_the_things
+    some_really_complicated_code
```

Comment:
Ben, is this even necessary?  Why do **you** **always** **overcomplicate things**?